### PR TITLE
MC-18097: Added api doc for custom ui settings

### PR DESCRIPTION
**Pull request by fyrnaga**
#### Changes made

- Added custom ui api docs

#### Related PRs
- []()

### CODE REVIEW

File: source/includes/administration/_custom_ui_settings.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+Required | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The configured custom UI settings' id.
+`organization.id`<br/>*UUID* | The organization id that the custom UI settings are linked to. It cannot be changed.
+`enabled`<br/>*boolean* | If the custom ui redirection is enabled or not.
+
+Optional | &nbsp;
+---------- | -----------
+`customUIRootUrl`<br/>*string* | The base root that the user will be redirected to once the user is login. . Required if the custom ui is enabled.
```
Comment (evanluc) [inline source/includes/administration/_custom_ui_settings.md]:
```suggestion
`customUIRootUrl`<br/>*string* | The base root that the user will be redirected to once the user is login. Required if the custom ui is enabled.
```

**Review by evanluc — APPROVED** [review]

File: source/includes/administration/_custom_ui_settings.md (hunk 1 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: 749825adbae8da79ed2563e9358614bf437a03a5

File: source/includes/administration/_custom_ui_settings.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+
+#### Find custom ui settings associated to an organization
+
+`GET /reseller/settings/custom_ui/find?organizationId=:id`
+
+Retrieve the custom ui settings associated to an organization. If the `organizationId` is omitted, the authenticated user's organization will be used.
+
+```shell
+# Retrieve the custom ui settings
+curl "https://cloudmc_endpoint/api/rest/reseller/settings/custom_ui/find?organizationId=10572c3d-16e5-450f-8af8-a01e50dc52d4" \
```
Comment (HassanHaidar) [inline source/includes/administration/_custom_ui_settings.md]: i think this should be https://cloudmc_endpoint/api/v2/reseller/...

File: source/includes/administration/_custom_ui_settings.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The configured custom UI settings' id.
+`organization.id`<br/>*UUID* | The organization id that the custom UI settings are linked to. It cannot be changed.
+`version`<br/>*integer* | The custom UI settings version.
+`enabled`<br/>*boolean* | If the custom ui redirection is enabled or not.
+`customUIRootUrl`<br/>*string* | The base root that the user will be redirected to once the user is login.
+`domainToken`<br/>*string* | The domain name that will be used in the authentication cookie. The cookie must be acceptable by both the reseller organization custom domain and the url provided in this configuration.
+`applyTags`<br/>*Array[string]* | The list organization tags that will redirect the user to the custom ui if they match.
```
Comment (HassanHaidar) [inline source/includes/administration/_custom_ui_settings.md]: list of organization tags

File: source/includes/administration/_custom_ui_settings.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+
+
+<!-------------------- GET CUSTOM UI SETTINGS -------------------->
+#### Retrieve custom UI settings
+
+`GET /reseller/settings/custom_ui/:id`
+
+```shell
+# Retrieve custom UI settings
+curl "https://cloudmc_endpoint/api/rest/reseller/settings/custom_ui/d785ffcb-9b03-478d-a49b-52a2ccedf1b8" \
```
Comment (HassanHaidar) [inline source/includes/administration/_custom_ui_settings.md]: same comment as previous for the url

File: source/includes/administration/_custom_ui_settings.md (hunk 2 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: you are correct... will change it

**Review by HassanHaidar — COMMENTED** [review]
Small notes like:
- "cloudmc_endpoint/api/v2/reseller" instead of  "cloudmc_endpoint/api/rest/reseller" 
- "list of organization tags" instead of "list organization tags"

File: source/includes/administration/_custom_ui_settings.md (hunk 4 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: will fix all of them

**Review by HassanHaidar — APPROVED** [review]
Looks good 👍

File: source/includes/administration/_custom_ui_settings.md (hunk 4 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: 28a54025d74ba155e7339ea0a8207d38ead6aea3

File: source/includes/administration/_custom_ui_settings.md (hunk 3 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: 28a54025d74ba155e7339ea0a8207d38ead6aea3

File: source/includes/administration/_custom_ui_settings.md (hunk 2 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: 28a54025d74ba155e7339ea0a8207d38ead6aea3

File: source/includes/administration/_custom_ui_settings.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
+### Custom UI Settings
+
+<!-------------------- FIND CUSTOM UI SETTINGS -------------------->
+
+#### Find custom ui settings associated to an organization
+
+`GET /reseller/settings/custom_ui/find?organizationId=:id`
+
+Retrieve the custom ui settings associated to an organization. If the `organizationId` is omitted, the authenticated user's organization will be used.
```
Comment (patrickspensieri) [inline source/includes/administration/_custom_ui_settings.md]: To be consistent with rest of docs we can capitalize **UI** here

File: source/includes/administration/_custom_ui_settings.md [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+```
+
+Retrieve the custom UI settings associated to an organization.
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The configured custom UI settings' id.
+`organization.id`<br/>*UUID* | The organization id that the custom UI settings are linked to. It cannot be changed.
+`version`<br/>*integer* | The custom UI settings version.
+`enabled`<br/>*boolean* | If the custom ui redirection is enabled or not.
```
Comment (patrickspensieri) [inline source/includes/administration/_custom_ui_settings.md]: Same here, **UI** should be capitalized for consistency

File: source/includes/administration/_custom_ui_settings.md [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+`customUIRootUrl`<br/>*string* | The base root that the user will be redirected to once the user is login.
```
Comment (patrickspensieri) [inline source/includes/administration/_custom_ui_settings.md]: Should be **is logged in**

File: source/includes/administration/_custom_ui_settings.md [hunk 8]
```diff
@@ -0,0 +1,246 @@
...
+`domainToken`<br/>*string* | The domain name that will be used in the authentication cookie. The cookie must be acceptable by both the reseller organization custom domain and the url provided in this configuration.
+`applyTags`<br/>*Array[string]* | The list of organization tags that will redirect the user of sub-organizations to the custom ui if they match.
+`denyTags`<br/>*Array[string]* | The list of organization tags that will not redirect the user of sub-organizations to the custom ui if they match.
```
Comment (patrickspensieri) [inline source/includes/administration/_custom_ui_settings.md]: Same here, should be custom **UI** to be consistent (and in a few other places in other sections)

**Review by patrickspensieri — APPROVED** [review]

File: source/includes/administration/_custom_ui_settings.md (hunk 5 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: e1fe79e8e800d38760fbeeac4d2ed4a17474aa49

File: source/includes/administration/_custom_ui_settings.md (hunk 6 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: e1fe79e8e800d38760fbeeac4d2ed4a17474aa49

File: source/includes/administration/_custom_ui_settings.md (hunk 7 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: e1fe79e8e800d38760fbeeac4d2ed4a17474aa49

File: source/includes/administration/_custom_ui_settings.md (hunk 8 above)
Comment (fyrnaga) [reply source/includes/administration/_custom_ui_settings.md]: e1fe79e8e800d38760fbeeac4d2ed4a17474aa49

**Pull request merged**